src/SourceBuild/content/eng/tools/BinaryToolKit/RemoveBinaries.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Extensions.Logging;
+
+namespace BinaryToolKit;
+
+public class RemoveBinaries
```

Comment:
Why is this an instance class if it doesn't store any instance data? I would either make it static or a true singleton. Looking at what this class does, this should probably not be an instance class and just a static helper method.

Reply:
> Consider following the provider pattern here and naming this RemoveBinariesProvider : IRemoveBinariesProvider.

While I agree that this design pattern is the most technically correct way to go, I feel like it is a bit heavy for the functionality of this tool. This tool is small, so I think that using a static logger and static classes are the way to go.

I did take your suggestion on making the class static since it doesn't store any instance data!